rpc_jobs/build_summary.yml

```diff
@@ -15,24 +15,9 @@
     dsl: |
       library "rpc-gating@${RPC_GATING_BRANCH}"
       common.globalWraps(){
+        String RE_JOB_REPO_NAME = "rpc-gating"
         dir("rpc-gating/scripts/build_summary"){
-          withCredentials([
-            usernamePassword(
-              credentialsId: "dockerhubrpcjirasvc",
-              usernameVariable: "DOCKERHUB_USER",
-              passwordVariable: "DOCKERHUB_PASSWORD"
-            )
-          ]){
-            sh """#!/bin/bash -xeu
-              # add docker registry creds.
-              # using creds in jenkins global config along with image.push()
-              # doesn't seem to work :(
-              # ok to use single quotes as the \$ is interpreted by groovy not bash
-              docker login --password '${env.DOCKERHUB_PASSWORD}' --username '${env.DOCKERHUB_USER}'
-              docker build . -t "${DOCKER_IMAGE}"
-              docker push "${DOCKER_IMAGE}"
-            """
-          }
+          docker_utils.pullOrBuild(DOCKER_IMAGE)
```

Comment:
Why not use the repo sha as the tag here?

Reply:
I misread `DOCKER_IMAGE` and thought it included the tag. I'll need to fix this.